src/zenml/integrations/aws/orchestrators/sagemaker_orchestrator.py

```diff
@@ -215,6 +218,23 @@ def prepare_or_run_pipeline(
             )
             logger.info("Pipeline completed successfully.")
 
+    def _get_region_name(self) -> str:
+        """Returns the AWS region name.
+
+        Returns:
+            The region name.
+
+        Raises:
+            RuntimeError: If the region name cannot be retrieved.
+        """
+        try:
+            return cast(str, sagemaker.Session().boto_region_name)
+        except Exception as e:
```

Comment:
Should this be such a hard error? Its only used to get the orchestrator link right? Maybe we can raise a warning instead of maybe catch it in get_pipeline_run_metadata in line 249 to not fail `hard`

Reply:
You're right. I'll fix this.